### ISSUE TITLE
creating a contributions page for the webiste

### ISSUE THREAD

**Issue by 1Shubham7** [1Shubham7/code-buddy#10]

**Comment by Prakashd20** [comment]
hey @1Shubham7 ,
could contribute for your website, can you assigned me for the same.

**Comment by Prakashd20** [comment]
@1Shubham7 you just need a contribution page or need to add-on something?

**Comment by 1Shubham7** [comment]
yup, we have a Contribution.md file. Rename it to CONTRIBUTIONS.md and add content to it to make it better.

**Comment by 1Shubham7** [comment]
Take your time bro. No worries, all the best for your exams. do contribute after that.

**Comment by Prakashd20** [comment]
would my contribution counted for kwoc on working with you on this project?

**Comment by 1Shubham7** [comment]
I guess yes since this project is under KWOC. Please confirm this from the website.
